terraform/concourse/concourse.tf

```diff
@@ -159,7 +159,10 @@ resource "terraform_data" "provision_concourse" {
     command = <<-CMD
       gcloud compute ssh ${var.hostname} --zone ${var.zone} --tunnel-through-iap --command "\
       sudo /concourse/scripts/install.sh ${var.docker_ver} ${var.data_disk} ;\
-      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert}" --verbosity=error
+      sudo /concourse/scripts/start.sh ${var.hostname}.${var.dns_zone_fqdn} ${var.force_new_cert}"
```

Comment:
end line with ;\ so that it doesn't break ssh command

Reply:
Yup, thanks I missed it :)